src/execution_plan/ops/op_node_by_label_scan.c

```diff
@@ -39,24 +54,33 @@ OpBase *NewNodeByLabelScanOp(const ExecutionPlan *plan, const QGNode *n) {
 	return (OpBase *)op;
 }
 
-static inline void _ConstructIterator(NodeByLabelScan *op) {
+void NodeByLabelScanOp_SetIDRange(NodeByLabelScan *op, UnsignedRange *id_range) {
+	op->id_range = UnsignedRange_New();
+	memcpy(op->id_range, id_range, sizeof(UnsignedRange));
+	op->minId = op->id_range->include_min ? op->id_range->min : op->id_range->min + 1;
+	op->op.type = OpType_NODE_BY_LABEL_AND_ID_SCAN;
+	op->op.name = "Node By Label and ID Scan";
```

Comment:
having it allocated in advance is wasteful IMO. Since I don't want to replace the op's `minID` and `maxID` with this range object, it might remain unused, so I want to allocate on demand.